build.gradle

```diff
@@ -18,17 +18,13 @@
 buildscript {
   repositories {
     jcenter()
-  }
-
-  // TODO: is this the right way to incluse gradle plugins?
-  repositories {
-    repositories {
-      maven {
-        // For license plugin gradle-rat.
-        url 'http://dl.bintray.com/content/netflixoss/external-gradle-plugins/'
-        // For avro compiler plugin gradle-avro-plugin
-        url 'https://plugins.gradle.org/m2/'
-      }
+    maven {
+      // For license plugin gradle-rat.
+      url 'http://dl.bintray.com/content/netflixoss/external-gradle-plugins/'
+    }
+    maven {
+      // For avro compiler plugin gradle-avro-plugin
+      url 'https://plugins.gradle.org/m2/'
     }
   }
```

Comment:
- Multiple repositories can be listed in one block, breaking them into 2 blocks is unnecessary.
- Repository blocks shouldn't be nested
- Each Maven repository block should contain only 1 URL
- The repository at the first Maven URL above is unnecessary: the rat plugin jar is available on jcenter (and it was likely being overwritten by the second URL anyway)

I believe that's it